### DISCUSSION TITLE
How do I change the header background color only?

### DISCUSSION THREAD

**Discussion by yoe88** [nick-keller/react-datasheet-grid#211] [Q&A] [answered]
[https://github.com/nick-keller/react-datasheet-grid/blob/master/src/style.css](url)

I saw the style document of the link above, but there was no variable that only changed the background color of the header.
**--dsg-cell-background-color** This variable changes all background colors.

If possible, can you create a variable for the header background color?

**Comment by nick-keller** [comment, marked as answer]
Hey @hijr88 ,
1. You can add a [custom class](https://react-datasheet-grid.netlify.app/docs/styling#custom-classes) to the header cell of a specific column using the [headerClassName](https://react-datasheet-grid.netlify.app/docs/api-reference/columns#headerclassname) option of a column.
2. Or you can add a style to all header cells using CSS: `.dsg-cell-header { background: red; }`
